aether-odk-module/aether/odk/api/xform_utils.py

```diff
@@ -235,16 +239,16 @@ def parse_xform_to_avro_schema(xml_definition, default_version=DEFAULT_XFORM_VER
     title, form_id, version = get_xform_data_from_xml(xml_definition)
     version = version or default_version
     # include version within name to identify different entity source
-    name = f'{form_id}_{version}'
+    name = project_name
```

Comment:
`name = project_name or f'{form_id}_{version}`  because project name can be blank :woman_shrugging:

Reply:
Should I make the argument optional?

Reply:
as you wish